lib/rspec/rails/view_rendering.rb

```diff
@@ -81,7 +81,11 @@ def append_view_path(new_path)
       private
 
         def _path_decorator(path)
-          EmptyTemplateResolver.new(path)
+          if path.is_a?(::ActionView::Resolver)
+            path
```

Comment:
PR #1535 switched form using `EmptyTemplatePathSetDecorator`, a subclass of `ActionView::Resolver`, to using `EmptyTemplateResolver`, a subclass of `ActionView::FileSystemResolver`. This means that when I have called `prepend_view_path` with a `Resolver` instance instead of a string/path argument, I run into this error: `path already is a Resolver class` from `FileSystemResolver#initialize`: https://github.com/rails/rails/blob/v4.2.5.1/actionview/lib/action_view/template/resolver.rb#L324-L329